### PR TITLE
V3.8.0

### PR DESCRIPTION
**Pull request by kagg-design**

### CODE REVIEW

**Comment by kagg-design** [comment]
@e271828- could you please review and merge?

File: assets/js/kagg-dialog.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
+// noinspection ES6ConvertVarToLetConst
+
+// eslint-disable-next-line no-var
+var kaggDialog = window.kaggDialog || ( function( document ) {
+	return {
+		defaults: {
+			title: 'Are you sure to continue?',
```
Comment (e271828-) [inline assets/js/kagg-dialog.js:7]:
```suggestion
			title: 'Do you want to continue?',
```

File: readme.txt [hunk 2]
```diff
@@ -540,6 +540,25 @@ Instructions for popular native integrations are below:
 
 == Changelog ==
 
+= 3.8.0 =
+* Added search of plugin and themes on the Integrations page.
+* Added toggling of sections on the General page.
+* Added new dialog on activation and deactivaion of plugin and themes.
```
Comment (e271828-) [inline readme.txt:546]:
```suggestion
* Added new dialog on activation and deactivation of plugin and themes.
```

File: readme.txt [hunk 3]
```diff
@@ -540,6 +540,25 @@ Instructions for popular native integrations are below:
...
+* Added selection of a new theme on deactivation of the current one.
+* Added 'backend' to optional Enterprise settings.
+* Added filter `hcap_api_host`, allowing to filter the API host.
+* Added filter `hcap_api_src`, allowing to filter the API source url with params.
+* Updated integration with Back In Stock Notifier.
+* Fixed Brevo plugin position on Integrations page.
```
Comment (e271828-) [inline readme.txt:552]:
```suggestion
* Fixed Brevo (formerly Sendinblue) plugin position on Integrations page.
```

**Review by e271828- — APPROVED** [review]

**Pull request merged**